src/extensions/default/bramble/lib/PostMessageTransport.js

```diff
@@ -111,7 +111,7 @@ define(function (require, exports, module) {
             // Not one of our custom events, re-trigger message event to LiveDev
             module.exports.trigger("message", [connId, msgObj.message]);
         } else if (msgObj.type === "connect") {
-            Browser.setListener();
+            //Browser.setListener();
```

Comment:
Let's actually remove this vs. just commenting out, so that we don't leave it in and have it get re-enabled some time later.  If we need to get this back, it's always there in git.